frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/vulnerabilities.jspf

```diff
@@ -70,7 +70,7 @@
                 '<select class="toplabelledInput" ' +
                 'id="verificationChangeSelectFor<sw360:out value='${vulnerability.externalId}'/>" ' +
                 'name="<portlet:namespace/><%=VerificationState.class%>" ' +
-                'onchange="changeVerification(this, `<sw360:out value='${vulnerability.externalId}'/>`,`${vulnerabilityVerifications.get(vulnerability.externalId).get(releaseId).ordinal()}`)" ' +
+                "onchange='changeVerification(this, &quot;<sw360:out value='${vulnerability.externalId}'/>&quot;,&quot;${vulnerabilityVerifications.get(vulnerability.externalId).get(releaseId).ordinal()}&quot;)' " +
```

Comment:
The patch is 
```
From b472d9e24edfd1ceb2b234ac042cd4fa2687e33a Mon Sep 17 00:00:00 2001
From: Alex Borodin <alex.borodin@evosoft.com>
Date: Thu, 4 May 2017 17:26:42 +0200
Subject: [PATCH] fix(ui): remove backticks from JS code to eliminate Liferay's
 minifier errors

---
 .../src/main/webapp/WEB-INF/tags/TrimLineBreaks.tag          | 12 ++++++++++++
 .../html/components/includes/components/vulnerabilities.jspf |  2 +-
 .../html/components/includes/releases/vulnerabilities.jspf   |  2 +-
 .../sw360-portlet/src/main/webapp/html/components/view.jsp   |  5 +++--
 .../html/projects/includes/projects/vulnerabilities.jspf     |  2 +-
 5 files changed, 18 insertions(+), 5 deletions(-)
 create mode 100644 frontend/sw360-portlet/src/main/webapp/WEB-INF/tags/TrimLineBreaks.tag

diff --git a/frontend/sw360-portlet/src/main/webapp/WEB-INF/tags/TrimLineBreaks.tag b/frontend/sw360-portlet/src/main/webapp/WEB-INF/tags/TrimLineBreaks.tag
new file mode 100644
index 00000000..3570f63a
--- /dev/null
+++ b/frontend/sw360-portlet/src/main/webapp/WEB-INF/tags/TrimLineBreaks.tag
@@ -0,0 +1,12 @@
+<%--
+  ~ Copyright Siemens AG, 2017. Part of the SW360 Portal Project.
+  ~
+  ~ All rights reserved. This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Public License v1.0
+  ~ which accompanies this distribution, and is available at
+  ~ http://www.eclipse.org/legal/epl-v10.html
+  ~
+  ~ This tag file MUST NOT have any line breaks outside of this multiline comment.
+  ~ It is surely possible to implement it in Java to avoid having this weird restriction,
+  ~ but this way was perceived as quickest by me at the time.
+  --%><%@ attribute name="input" type="java.lang.String" required="true" %><%@ taglib prefix="c" uri="http://java.sun.com/jstl/core_rt"%><%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions" %><c:set var="newline" value='<%="\n"%>' />${fn:replace(input, newline, '')}
\ No newline at end of file
diff --git a/frontend/sw360-portlet/src/main/webapp/html/components/includes/components/vulnerabilities.jspf b/frontend/sw360-portlet/src/main/webapp/html/components/includes/components/vulnerabilities.jspf
index 8f1e9006..ec3cc09f 100644
--- a/frontend/sw360-portlet/src/main/webapp/html/components/includes/components/vulnerabilities.jspf
+++ b/frontend/sw360-portlet/src/main/webapp/html/components/includes/components/vulnerabilities.jspf
@@ -67,7 +67,7 @@
                     '<select class="toplabelledInput" ' +
                     'id="verificationChangeSelectFor${vulnerability.externalId}" ' +
                     'name="<portlet:namespace/><%=VerificationState.class%>" ' +
-                    'onchange="changeVerification(this, `${vulnerability.externalId}`,`${vulnerabilityVerifications.get(vulnerability.externalId).get(vulnerability.intReleaseId).ordinal()}`, `${vulnerability.intReleaseId}`)" ' +
+                    'onchange="changeVerification(this, \'${vulnerability.externalId}\',\'${vulnerabilityVerifications.get(vulnerability.externalId).get(vulnerability.intReleaseId).ordinal()}\', \'${vulnerability.intReleaseId}\')" ' +
                     'title="<sw360:out value='${vulnerabilityVerificationTooltips.get(vulnerability.externalId).get(vulnerability.intReleaseId)}'/>"'+
                     'style="min-width: 90px; min-height: 28px;">' +
                     <sw360:DisplayEnumOptions type="<%=VerificationState.class%>" selected="${vulnerabilityVerifications.get(vulnerability.externalId).get(vulnerability.intReleaseId)}" inQuotes="true"/> +
diff --git a/frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/vulnerabilities.jspf b/frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/vulnerabilities.jspf
index ba7fee68..93f4abc2 100644
--- a/frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/vulnerabilities.jspf
+++ b/frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/vulnerabilities.jspf
@@ -70,7 +70,7 @@
                 '<select class="toplabelledInput" ' +
                 'id="verificationChangeSelectFor<sw360:out value='${vulnerability.externalId}'/>" ' +
                 'name="<portlet:namespace/><%=VerificationState.class%>" ' +
-                'onchange="changeVerification(this, `<sw360:out value='${vulnerability.externalId}'/>`,`${vulnerabilityVerifications.get(vulnerability.externalId).get(releaseId).ordinal()}`)" ' +
+                'onchange="changeVerification(this, \'<sw360:out value='${vulnerability.externalId}'/>\',\'${vulnerabilityVerifications.get(vulnerability.externalId).get(releaseId).ordinal()}\')" ' +
                 'title="<sw360:out value='${vulnerabilityVerificationTooltips.get(vulnerability.externalId).get(releaseId)}'/>"'+
                 'style="min-width: 90px; min-height: 28px;">' +
                 <sw360:DisplayEnumOptions type="<%=VerificationState.class%>" selected="${vulnerabilityVerifications.get(vulnerability.externalId).get(releaseId)}" inQuotes="true"/> +
diff --git a/frontend/sw360-portlet/src/main/webapp/html/components/view.jsp b/frontend/sw360-portlet/src/main/webapp/html/components/view.jsp
index f4ca3fd4..c70fd7a6 100644
--- a/frontend/sw360-portlet/src/main/webapp/html/components/view.jsp
+++ b/frontend/sw360-portlet/src/main/webapp/html/components/view.jsp
@@ -1,5 +1,5 @@
 <%--
-  ~ Copyright Siemens AG, 2013-2016. Part of the SW360 Portal Project.
+  ~ Copyright Siemens AG, 2013-2017. Part of the SW360 Portal Project.
   ~ With modifications by Bosch Software Innovations GmbH, 2016.
   ~
   ~ All rights reserved. This program and the accompanying materials
@@ -248,11 +248,12 @@
         var result = [];
 
         <core_rt:forEach items="${componentList}" var="component">
+        <core_rt:set var="licenseCollectionTagOutput"><tags:DisplayLicenseCollection licenseIds="${component.mainLicenseIds}" scopeGroupId="${pageContext.getAttribute('scopeGroupId')}"/></core_rt:set>
         result.push({
             "DT_RowId": "${component.id}",
             "0": '<sw360:DisplayCollection value="${component.vendorNames}"/>',
             "1": "<a href='" + createDetailURLfromComponentId("${component.id}") + "' target='_self'><sw360:out value="${component.name}"/></a>",
-            "2": `<tags:DisplayLicenseCollection licenseIds="${component.mainLicenseIds}" scopeGroupId="${pageContext.getAttribute('scopeGroupId')}"/>`,
+            "2": "<tags:TrimLineBreaks input="${licenseCollectionTagOutput}"/>",
             "3": '<sw360:DisplayEnum value="${component.componentType}"/>',
             "4": "<a href='<portlet:renderURL ><portlet:param name="<%=PortalConstants.COMPONENT_ID%>" value="${component.id}"/><portlet:param name="<%=PortalConstants.PAGENAME%>" value="<%=PortalConstants.PAGENAME_EDIT%>"/></portlet:renderURL>'><img src='<%=request.getContextPath()%>/images/edit.png' alt='Edit' title='Edit'> </a>"
             + "<img src='<%=request.getContextPath()%>/images/Trash.png' onclick=\"deleteComponent('${component.id}', '<b>${component.name}</b>',${component.releaseIdsSize},${component.attachmentsSize})\"  alt='Delete' title='Delete'>"
diff --git a/frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf b/frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf
index 54890be4..c47cea84 100644
--- a/frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf
+++ b/frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf
@@ -70,7 +70,7 @@
                 '<select class="toplabelledInput" '+
                     'id="ratingChangeSelectFor${vulnerability.externalId}" '+
                     'name="<portlet:namespace/><%=VulnerabilityRatingForProject.class%>" '+
-                    'onchange="changeRating(this, `${vulnerability.externalId}`,`${vulnerabilityRatings.get(vulnerability.externalId).get(vulnerability.intReleaseId).ordinal()}`, `${vulnerability.intReleaseId}`)" '+
+                    'onchange="changeRating(this, \'${vulnerability.externalId}\',\'${vulnerabilityRatings.get(vulnerability.externalId).get(vulnerability.intReleaseId).ordinal()}\', \'${vulnerability.intReleaseId}\')" '+
                     'title="<sw360:out value='${vulnerabilityCheckstatusTooltips.get(vulnerability.externalId).get(vulnerability.intReleaseId)}'/>"'+
                     'style="min-width: 90px; min-height: 28px;">'+
                    <sw360:DisplayEnumOptions type="<%=VulnerabilityRatingForProject.class%>" selected="${vulnerabilityRatings.get(vulnerability.externalId).get(vulnerability.intReleaseId)}" inQuotes="true"/> +
-- 
2.12.2
```